packages/models-library/src/models_library/function_service/demo_units.py

```diff
@@ -15,11 +15,14 @@ def build_input(schema):
 
 
 # SEE https://github.com/hgrecco/pint/blob/master/pint/default_en.txt
+
 META = ServiceDockerData.parse_obj(
     {
         "integration-version": LATEST_INTEGRATION_VERSION,
         "key": f"{FUNCTION_SERVICE_KEY_PREFIX}/data-iterator/demo-units",
-        "version": "0.1.0",
+        "version": "0.2.0",
+        # CHANGELOG
+        # - 0.2.0: reverted order of first 5 outputs
```

Comment:
IDEA: reading this I would suggest to extend the model and actually add a changeling field, that would look better

Reply:
I would rather add a link to a repo or a file for that instead of adding more metadata